dm/syncer/schema.go

```diff
@@ -144,21 +142,21 @@ func (s *Syncer) OperateSchema(ctx context.Context, req *pb.OperateWorkerSchemaR
 		}
 
 		s.tctx.L().Info("flush table info", zap.String("table info", newSQL))
-		err = s.checkpoint.FlushPointsWithTableInfos(tcontext.NewContext(ctx, log.L()), []*filter.Table{sourceTable}, []*model.TableInfo{ti})
+		err = s.checkpoint.FlushPointsWithTableInfos(s.tctx.WithContext(ctx), []*filter.Table{sourceTable}, []*model.TableInfo{ti})
```

Comment:
For curiosity,  tidb prefers to put things in inner map of context, but dm put the logger with the context.
What's the principles of this stuff?